[SSL] New FAQ page on CAs and certs (#10178)
cloudflare/cloudflare-docs: content/ssl/reference/migration-guides/digicert-update/_index.md

{{<directory-listing>}}

This change will not impact existing certificates issued from DigiCert that Cloudflare is currently serving, but it will impact new certificate orders and renewals.

> Consider the [CAs and certificates FAQ](/ssl/edge-certificates/troubleshooting/ca-faq/) for commonly asked questions that you may have.

Reviewer: ```suggestion
Refer to [CAs and certificates FAQ](/ssl/edge-certificates/troubleshooting/ca-faq/) for commonly asked questions that you may have.
```
